server/routes/temporaryabsences/confirmTemporaryAbsenceAddedToRollController.ts

```diff
@@ -1,9 +1,27 @@
 import { RequestHandler } from 'express'
+import type { PrisonService } from '../../services'
```

Comment:
Might want to add `redirectIfDisabledMiddleware(config.confirmEnabled)` to the GET of this router